nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -56,16 +59,33 @@ class LazyNeMoIterator:
     Example::
 
         >>> cuts = lhotse.CutSet(LazyNeMoIterator("nemo_manifests/train.json"))
+
+    We allow attaching custom metadata to cuts from files other than the manifest via ``extra_fields`` argument.
+    In the example below, we'll iterate file "questions.txt" together with the manifest and attach each line
+    under ``cut.question`` using the field type ``text_iter``::
+
+        >>> cuts = lhotse.CutSet(LazyNeMoIterator(
+        ...     "nemo_manifests/train.json",
+        ...     extra_fields=[{"type": "text_iter", "name": "question", "path": "questions.txt"}],
```

Comment:
is there value in allowing to path to be a list of questions also ?

Reply:
I'm not sure I understood your comment - do you mean `"path": ["questions1.txt", "questions2.txt"]`? what would be the expected behavior?